app/utils/constants.tsx

```diff
@@ -0,0 +1,90 @@
+export const constants = {
+  contactEmail: "post@capraconsulting.no",
+  contactPhoneNumber: "45832632",
+  iconSize: 24,
```

Comment:
De andre konstantene representerer generell kontakt informasjon. `iconSize` representerer presentasjons detaljer, knyttet til Footeren. jeg mener den hører hjemme i `footer.tsx`

💭 
Om vi evt. når vi får behov for det, skal ha noe global config på presentasjons detaljer burde det kanskje være i en eget objekt. (eller definert i tailwind config?)

Reply:
Enig 🚀 [9926d44](https://github.com/capraconsulting/nettsiden/pull/7/commits/9926d44c896ae880205fb082a02e54a095d9d8ec)